Libraries/PermissionsAndroid/PermissionsAndroid.js

```diff
@@ -128,6 +135,15 @@ class PermissionsAndroid {
     }
     return Permissions.requestPermission(permission);
```

Comment:
Change flow return type from `Promise<boolean>` to `Promise<string>`

Will at least warn people using flow that this api has changed